testing/bdd/features/handover/handover_algorithm.feature

```diff
@@ -33,18 +33,17 @@ Feature: Handover algorithm
     And it has <authors_group_control_token2>
     And the document is generated
     And handover node processes document
-    Then handover node emits <emitted_documents> documents
+    Then the emitted documents have <authors_group_selected_sequence_identifiers>
     And the emitted documents belong to <sequence_identifier> and use consecutive sequence numbering from 1
     And the emitted documents have <authors_group_identifier> and they specify a token
 
     Examples:
-    | sequence_identifier1 | sequence_number1 | authors_group_control_token1 | sequence_identifier2 | sequence_number2 | authors_group_control_token2 | emitted_documents |
-    | seq1                 | 1                | 1                            | seq2                 | 1                | 2                            | 2                 |
-    | seq1                 | 1                | 2                            | seq2                 | 1                | 3                            | 2                 |
-    | seq1                 | 1                |                              | seq2                 | 1                | 1                            | 1                 |
+    | sequence_identifier1 | sequence_number1 | authors_group_control_token1 | sequence_identifier2 | sequence_number2 | authors_group_control_token2 | authors_group_selected_sequence_identifiers |
+    | seq1                 | 1                | 1                            | seq2                 | 1                | 2                            | seq1,seq2                                   |
```

Comment:
I would like to add back in the `emitted_documents` count test. I will do that. There are also some other test cases that can be added.